functest/main_test.go

```diff
@@ -360,11 +360,21 @@ func setup() error {
 		return errors.Wrap(err, "[ setup ] could't start insolard: ")
 	}
 	fmt.Println("[ setup ] insolard was successfully started")
-	time.Sleep(60 * time.Second)
-	err = setInfo()
+
+	NUM_ATTEMPTS := 60
```

Comment:
You do not want to do UPPERCASE constants in golang :)

Reply:
it's not me :/